### ISSUE TITLE
filter for lobby list

### ISSUE THREAD

**Issue by Unbaloav** [ThePandemoniumInstitute/botc-release#142]
I would love to be able to automatically filter out lobbies that for example are in a language I don't speak or games that are streamed.

**Comment by bra1n** [comment]
Yup, this is on the to-do list, but only once there are enough public lobbies so that filtering is actually feasible. :)
